cdap-docs/admin-manual/source/installation/cloudera.rst

```diff
@@ -398,20 +407,29 @@ Add Service Wizard: Completion Page
    :align: center
    :class: bordered-image
 
-   **Add Service Wizard, Page 7:** Congratulations screen, though there is still work to be done.
+   **Add Service Wizard, Page 6:** Congratulations screen, though there is still work to be done.
 
 
 Cluster Home Page: Status Tab
 -----------------------------
 
-.. figure:: ../_images/cloudera/cloudera-csd-09.png
+.. figure:: ../_images/cloudera/cloudera-csd-09a.png
    :figwidth: 100%
    :width: 800px
    :align: center
    :class: bordered-image
 
    **Cluster Home Page, Status Tab:** Showing all CDAP services running. *Gateway* is not an actual service.
 
+.. figure:: ../_images/cloudera/cloudera-csd-09b.png
+   :figwidth: 100%
+   :width: 800px
+   :align: center
+   :class: bordered-image
+
+   **Cloudera Manager Home Page:** Showing CDAP installed on the cluster as a service.
```

Comment:
it's a little wierd that the 09a screenshot shows CDAP running in the detail service view, but then this 09b screenshot of the cluster view shows it stopped and reconfiguration needed.  Ideally, we'd show a healthy CDAP here